### PR TITLE
✅More visual diff tests for AMPHTML ads in both FIE & inabox

### PR DESCRIPTION
**Pull request by powerivq**
Including both FIE and inabox type of A4A components, w/ two sample ads each.

### CODE REVIEW

File: examples/visual-tests/amp-ad/amp-fie-doubleclick.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const {verifySelectorsVisible} = require('../../../build-system/tasks/visual-diff/helpers');
+
+module.exports = {
```
Comment (lannka) [inline examples/visual-tests/amp-ad/amp-fie-doubleclick.js]: seems to be standard boilerplate code. but could you add some comments just for education purpose. thx

File: examples/visual-tests/amp-ad/amp-fie-doubleclick-image.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  <link rel="canonical" href="https://amp.dev/documentation/examples/introduction/hello_world/index.html">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <script async custom-element="amp-ad" src="https://cdn.ampproject.org/v0/amp-ad-0.1.js"></script>
+</head>
+<body>
+<h1>AMP FIE Doubleclick ad test 2</h1>
```
Comment (lannka) [inline examples/visual-tests/amp-ad/amp-fie-doubleclick-image.html]: AMPHTML ad - FIE - /amp-ad/doubleclick/image.html

File: examples/visual-tests/amp-ad/amp-fie-doubleclick-text.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  <link rel="canonical" href="https://amp.dev/documentation/examples/introduction/hello_world/index.html">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <script async custom-element="amp-ad" src="https://cdn.ampproject.org/v0/amp-ad-0.1.js"></script>
+</head>
+<body>
+<h1>AMP FIE Doubleclick ad test 1</h1>
```
Comment (lannka) [inline examples/visual-tests/amp-ad/amp-fie-doubleclick-text.html]: same here

File: examples/visual-tests/amp-ad/amp-fie-doubleclick.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
+/**
```
Comment (lannka) [inline examples/visual-tests/amp-ad/amp-fie-doubleclick.js]: i think the test is doubleclick irrelevant? let's rename the file

/amp-ad/fie.js

File: examples/visual-tests/amp-ad/amp-fie-doubleclick-image.html [hunk 5]
```diff
@@ -0,0 +1,24 @@
+<!doctype html>
```
Comment (lannka) [inline examples/visual-tests/amp-ad/amp-fie-doubleclick-image.html]: file naming: /amp-ad/fie-image.html

**Comment by powerivq** [comment]
FYI I renamed the names of visual tests so it can be more easily filtered `gulp visual-diff --grep=amp-ad`

**Pull request merged**